main.py

```diff
@@ -79,6 +84,50 @@ def show_top_spending_category(df):
 
 
 
+def visualize_monthly_spending_trend(df):
+    print("--- Visualize Monthly Spending Trend ---")
+    # Get expense data, using copy to avoid affecting original df
+    expense_df = df[df["Type"] == "Expense"].copy()
+    # Convert Date column to datetime format
+    expense_df["Date"] = pd.to_datetime(expense_df["Date"])
+    # Filter month and add column called as Month
+    expense_df["Month"] = expense_df["Date"].dt.to_period("M")
+    # Group by Month to sum the spending
+    monthly_spending = expense_df.groupby("Month")["Amount"].sum().reset_index()
+    # For plotting, have to convert month back to datetime
+    monthly_spending["Month"] = monthly_spending["Month"].dt.to_timestamp()
+
+
+    # Line chart : Monthly Spending
+    plt.plot(monthly_spending["Month"], monthly_spending["Amount"] , marker='o')
+    plt.title('Monthly Spending Trend')
+    plt.xlabel('Month')
+    plt.ylabel('Total Spending ($)')
+    plt.xticks(rotation=45)
+    plt.grid()
+    plt.show()
```

Comment:
Does this line chart include all months data? Because this function should visialize one month data, so you should select one month data, for example latest month, etc.